data/events/writethedocs-eu-2018.json

```diff
@@ -0,0 +1,20 @@
+{
+    "accessibility": "",
+    "code_of_conduct": "http://www.writethedocs.org/code-of-conduct/",
+    "event_end": "2018-09-13",
+    "event_start": "2018-09-11",
+    "facebook": "",
+    "languages": [
+        "English"
+    ],
+    "location": {
+        "city": "Prague",
+        "country": "Czech Republic",
+        "state": ""
+    },
+    "name": "Write The Docs EU",
+    "tags": [],
+    "twitter": "writethedocs",
+    "videos_url": "http://pyvideo.org/",
```

Comment:
Leave this entry empty. pyvideo.org is for pycon/python conferences.

Reply:
I read over this line - another great catch!